### PR TITLE
Replace reach router with react router

### PR DESCRIPTION
**Pull request by BenBrownCreative**
not sure if this will solve the refresh issue with profile pages in production. The routing should handle the same as it did with reach router. some small differences.

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -52,13 +52,27 @@ ReactDOM.render(
...
-          <Router primary={false}>
-            <Home path="/" title="home page" />
-            <Event path="/event" />
-            <Profile path="/profile/:username" />
-            <Auth path="/sign-in" />
-            <NotFound default />
+          <Router>
+            <Header />
+            {/* A <Switch> looks through its children <Route>s and
+            renders the first one that matches the current URL. */}
```
Comment (meyer-mcmains) [inline client/src/index.js:58]:
```suggestion
```

File: client/src/pages/Home/index.js [hunk 2]
```diff
@@ -13,13 +14,13 @@ const Wrapper = styled.div`
   padding: 10vh 3vw;
 `;
 
-const Home = () => {
+const Home = props => {
```
Comment (meyer-mcmains) [inline client/src/pages/Home/index.js:17]:
```suggestion
const Home = ({ history }) => {
```

File: client/src/pages/Home/index.js [hunk 3]
```diff
@@ -13,13 +14,13 @@ const Wrapper = styled.div`
...
   const handleAddOption = e => {
     e.preventDefault();
     const username = e.target.elements.bggUsername.value.trim();
 
     localStorage.setItem('username', `${username}`);
-    navigate(`../profile/${username}`);
+    props.history.push(`../profile/${username}`);
```
Comment (meyer-mcmains) [inline client/src/pages/Home/index.js:23]:
```suggestion
    history.push(`../profile/${username}`);
```

**Review by meyer-mcmains — CHANGES_REQUESTED** [review]

File: client/src/pages/Home/index.js (hunk 2 above)
Comment (BenBrownCreative) [reply client/src/pages/Home/index.js]: good call

**Comment by BenBrownCreative** [comment]
i made the changes requested.

**Review by meyer-mcmains — APPROVED** [review]

**Pull request merged**